build-system/tasks/runtime-test/index.js

```diff
@@ -506,6 +512,12 @@ async function runTests() {
       message += '\n';
       console./*OK*/log('\n');
       log(message);
+    }).on('run_complete', (browsers, results) => {
+      if (results.error) {
```

Comment:
I think you need to report an error here too:

https://github.com/ampproject/amphtml/blob/1c54451b82cb9048b566d33ea3ad99e3a820183f/build-system/tasks/runtime-test/index.js#L488-L496

Reply:
`run_complete` is called once after `browser_complete`, so for Sauce Labs tests that run on multiple browsers this will create multiple reports. I can add ` || results.total == 0` to the above condition - would that make sense?

Reply:
As written, `run_complete` won't be called because of the `process.exit(1)` inside `browser_complete`.